### ISSUE TITLE
About unanswerable questions

### ISSUE THREAD

**Issue by YuxuanJiang1** [shmsw25/AmbigQA#22]
It seems unanswerable questions are not considered, is that right?

**Comment by shmsw25** [comment]
Yes! We don't have any unanswerable questions. If we find some questions unanswerable during annotation, we excluded them in the dataset.

**Comment by YuxuanJiang1** [comment]
> Yes! We don't have any unanswerable questions. If we find some questions unanswerable during annotation, we excluded them in the dataset.

Thanks for your quick reply!

Btw, do you have ideas on how to using your work on handling unanswerable questions in squad2.0 or natural questions? I'm trying to do some work on that.
